src/tcgwars/logic/impl/gen4/CallOfLegends.groovy

```diff
@@ -786,598 +452,215 @@ public enum CallOfLegends implements LogicCardInfo {
-        return basic (this, hp:HP070, type:FIGHTING, retreatCost:1) {
-          weakness P
-          move "Detect", {
-            text "Flip a coin. If heads, prevent all effects of attacks, including damage, done to Hitmonchan during your opponent’s next turn."
+      case TYPHLOSION_35:
+        return copy(HeartgoldSoulsilver.TYPHLOSION_32, this);
+      case TYROGUE_36:
+        return copy(HeartgoldSoulsilver.TYROGUE_33, this);
+      case URSARING_37:
+        return copy(Unleashed.URSARING_27, this);
+      case WEEZING_38:
+        return copy(HeartgoldSoulsilver.WEEZING_34, this);
+      case ZANGOOSE_39:
+        return basic (this, hp:HP080, type:COLORLESS, retreatCost:2) {
+          weakness F
+          move "Swords Dance", {
+            text "During you next turn, Zangoose’s Lost Claw attack’s base damage is 80."
             energyCost C
             attackRequirement {}
             onAttack {
-              damage 0
+              increasedBaseDamageNextTurn("Lost Claw", hp(50))
             }
           }
-          move "Sky Uppercut", {
-            text "30 damage. This attack’s damage isn’t affected by Resistance."
-            energyCost F, C
+          move "Lost Claw", {
+            text "30 damage. Choose 1 card from your opponent’s hand without looking and put it in the Lost Zone."
+            energyCost C, C, C
             attackRequirement {}
             onAttack {
               damage 30
-              dontApplyResistance()
+              opp.hand.shuffledCopy().select(hidden: true, "Choose 1 card to put in the Lost Zone.").moveTo(opp.lostZone)
             }
           }
 
         };
-      case HITMONLEE_58:
-        return basic (this, hp:HP080, type:FIGHTING, retreatCost:2) {
-          weakness P
-          move "Kick", {
-            text "20 damage. "
-            energyCost F
-            attackRequirement {}
-            onAttack {
-              damage 0
+      case BAYLEEF_40:
+        return copy(HeartgoldSoulsilver.BAYLEEF_35, this);
+      case CROCONAW_41:
+        return copy(HeartgoldSoulsilver.CROCONAW_38, this);
+      case DONPHAN_42:
+        return copy(HeartgoldSoulsilver.DONPHAN_40, this);
+      case FLAAFFY_43:
+        return copy(HeartgoldSoulsilver.FLAFFY_42, this);
+      case FLAREON_44:
+        return copy(Undaunted.FLAREON_26, this);
+      case JOLTEON_45:
+        return copy(Undaunted.JOLTEON_28, this);
+      case MAGBY_46:
+        return copy(Triumphant.MAGBY_41, this);
+      case MIME_JR__47:
+        return basic (this, hp:HP030, type:PSYCHIC, retreatCost:0) {
+          pokeBody "Sweet Sleeping Face", {
+            text "As long as Mime Jr. is Asleep, prevent all damage done to Mime Jr. by attacks."
+            delayedA {
+              before APPLY_ATTACK_DAMAGES, {
+                bg.dm().each {
+                  if(self.isSPC(ASLEEP) && it.to == self && it.dmg.value && it.notNoEffect) {
+                    bc "Sweet Sleeping Face: Damage was prevented"
```

Comment:
```suggestion
                    bc "$thisAbility prevents damage"
```
This can help make it easier if this is referenced for a copy/paste or as a candidate for a new static method. Also, most of the strings for these types of effects are written in this format, so best to try to keep it consistent.